src/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import "./navbar.css";
+
+const Navbar = () => {
+  const [showNav, setShowNav] = useState(" ");
+  const hamburgerClick = () => {
+    if (showNav === " ") setShowNav("mobile-navbar-view");
+    if (showNav === "mobile-navbar-view") {
+      setShowNav(" ");
+    }
+  };
+  return (
+    <div className="nav">
+      <nav className="navbar li-shadow">
+        <div className="nav-brand">
+          <span href="" className="logo-a">
+            <img className="logo-s" src="/images/logo.png" alt="" />
+          </span>
+        </div>
+        <div onClick={hamburgerClick} className="nav-hamburger">
+          <i className="fas fa-bars" />
+        </div>
+        <div className="nav-menu">
+          <div className="nav-start">
+            <span className="nav-item link">Dashboard</span>
+            <span className="nav-item link">Portfolio</span>
+            <span href="pages/searchpage.html" className="nav-item link">
+              Search
+            </span>
+          </div>
+          <div className="nav-end">
+            <button className="dark-mode btn-icon nav-icons m-x-1">
+              <i className="fas fa-moon" />
+            </button>
+            <button className="btn-secondary nav-btn btn-small">Sign up</button>
+            <button className="btn-primary nav-btn btn-small">Log in</button>
+          </div>
+        </div>
+      </nav>
+
+      <div className={`mobile-navbar ${showNav} text-center flex-c-w`}>
```

Comment:
Class can be added conditionally here `${showNav ? 'className'  : null }`

Reply:
Yes , thanks for the suggestion